test/fixtures/rules/css/all-css-class-errors.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
I found the tests a bit overwhelming and somewhat repetitive. I think we can make them a lot simpler by just having two test files.

```
test/fixtures/rules/css-modules
  | - undefined.js
  |- unused.js
```

```js
// undefined.js
import React from 'react';
import styles from './test.css';

// `css-modules/no-undef-class` - Alerts whenever a CSS class is not being used
// ---------------------------------------------------------------------
// Bad
(function () {
    const div = <div className={ styles.undefClass } />;
})();
// Good
(function () {
    const div = <div className={ styles.usedClass } />;
})();
```
The `unused.js` file would also follow this pattern.

Reply:
Removed old Tests and created new ones based on the template above